Jenkinsfile

```diff
@@ -10,7 +10,7 @@ def masterBranchName = 'master'
 def isMasterBranch = env.BRANCH_NAME == masterBranchName
 def developBranchName = 'develop'
 def isDevelopBranch = env.BRANCH_NAME == developBranchName
-def latestStableBranchName = 'stable/1.0'
+def latestStableBranchName = 'stable/1.1'
```

Comment:
Can we use wildcard markers in here?  Like `stable/1.*` or like a regex `stable/1\.[0-9]+`

Reply:
I would rather not, this would assign all wildcard matches the same CRON schedule.